boards/arm/usb_kw24d512/Makefile.board

```diff
@@ -0,0 +1,7 @@
+FLASH_SCRIPT = openocd.sh
+DEBUG_SCRIPT = openocd.sh
```

Comment:
You should be able to swap the mbed firmware with the jlink firmware.

Can you please add a link to your OpenOCD patch in the board doc?